Source/Details/ASAbstractLayoutController.h

```diff
@@ -20,6 +22,8 @@ AS_EXTERN CGRect CGRectExpandToRangeWithScrollableDirections(CGRect rect, ASRang
 
 @interface ASAbstractLayoutController : NSObject <ASLayoutController>
 
++ (std::vector<std::vector<ASRangeTuningParameters>>)defaultTuningParameters;
```

Comment:
Since this is a public header, I'm a bit worried about exposing Obj-C++ here since it's going to cause a cascade effect on every header file that imports it and every impl file that imports any of these headers. We're already seeing the effect in this PR.

How about we expose a Obj-C method here that converts the Obj-C data structure into a vector? Then maybe expose this Obj-C++ version in a **framework private** header, in case we want to mitigate any perf concern regarding the conversion.